File: catkin_ws/src/vel_pkg/src/vel_node.cpp
#include <ros/ros.h>
#include <geometry_msgs/Twist.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "vel_node");
    ros::NodeHandle nh;

    ros::Publisher vel_pub = nh.advertise<geometry_msgs::Twist>("/cmd_vel", 10);

    ros::Rate loop_rate(10);
    geometry_msgs::Twist vel_msg;
    while (ros::ok())
    {
        vel_msg.linear.x = 0.5;
        vel_msg.linear.y = 0.0;
        vel_msg.linear.z = 0.0;

        vel_msg.angular.x = 0.0;
        vel_msg.angular.y = 0.0;
        vel_msg.angular.z = 0.5;

        vel_pub.publish(vel_msg);
        ros::spinOnce();
        loop_rate.sleep(); 
    }
    return 0;
}

File: catkin_ws/src/vel_pkg/scripts/vel_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from geometry_msgs.msg import Twist

if __name__ == '__main__':
    rospy.init_node('vel_node_py')
    rospy.logwarn('[vel_node_py] init_node success !')

    pub = rospy.Publisher('cmd_vel', Twist, queue_size=10)
    rospy.logwarn('[vel_node_py] publisher success !')

    vel_msg = Twist()
     
    vel_msg.linear.x = 0.5
    vel_msg.linear.y = 0.0
    vel_msg.linear.z = 0.0
    vel_msg.angular.x = 0.0
    vel_msg.angular.y = 0.0
    vel_msg.angular.z = 0.5

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        pub.publish(vel_msg)
        rate.sleep()